samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
Please give proper name to this png:
5.Off356ConnCard.png

5.Office365ConnectorCard.png

Reply:
fixed